containers/release-candidate/daffodil-release-candidate

```diff
@@ -298,7 +298,7 @@ case $PROJECT_REPO in
     echo "Building Convenience Binaries..."
     mkdir -p $DAFFODIL_RELEASE_DIR/bin/
     yarn install
-    yarn build
+    yarn package
```

Comment:
Looks like we can remove `yarn install`? Doesn't seem to be needed anymore. I think that was only needed for the os thing which has since been removed/replaced.

Reply:
Oh yeah that is true because the packaging script does run it